src/Features/Core/Portable/Snippets/SnippetProviders/AbstractConsoleSnippetProvider.cs

```diff
@@ -139,10 +144,17 @@ protected override ImmutableArray<SnippetPlaceholder> GetPlaceHolderLocationsLis
             return openParenToken;
         }
 
-        protected override SyntaxNode? FindAddedSnippetSyntaxNode(SyntaxNode root, int position, ISyntaxFacts syntaxFacts)
+        private static async Task<INamedTypeSymbol?> GetSymbolFromMetaDataNameAsync(Document document, CancellationToken cancellationToken)
+        {
+            var compilation = await document.Project.GetRequiredCompilationAsync(cancellationToken).ConfigureAwait(false);
+            var symbol = compilation.GetBestTypeByMetadataName(typeof(Console).FullName!);
+            return symbol;
+        }
+
+        protected override SyntaxNode? FindAddedSnippetSyntaxNode(SyntaxNode root, int position, Func<SyntaxNode?, bool> IsCorrectContainer)
```

Comment:
```suggestion
        protected override SyntaxNode? FindAddedSnippetSyntaxNode(SyntaxNode root, int position, Func<SyntaxNode?, bool> isCorrectContainer)
```

This actually confused me, because I was looking for the `IsCorrectContainer` method, and didn't see the parameter :P